src/orders/components/OrderRefundDatagrid/OrderRefundDatagrid.tsx

```diff
@@ -54,22 +62,32 @@ export const OrderRefundDatagrid: React.FC<OrderRefundDatagridProps> = ({
     intl,
   });
 
-  const getMenuItems = React.useCallback(
-    index => [
+  const getMenuItems = React.useCallback(index => {
+    const refund = grantedRefunds[index];
+    const isEditable = isRefundEditable(refund);
+
+    return [
       {
         label: "",
-        Icon: (
-          <Link
-            to={orderGrantRefundEditUrl(orderId, grantedRefunds[index]?.id)}
-          >
+        Icon: isEditable ? (
+          <Link to={orderGrantRefundEditUrl(orderId, refund?.id)}>
             <EditIcon />
           </Link>
+        ) : (
+          <Tooltip>
+            <Tooltip.Trigger>
+              <EditIcon color="defaultDisabled" />
+            </Tooltip.Trigger>
+            <Tooltip.Content>
+              <Tooltip.Arrow />
+              <FormattedMessage {...refundGridMessages.notEditable} />
```

Comment:
suggestion: I think we should differentiate between status PENDING and SUCCESSFUL and create separate tooltip messages for these cases:
- This refund has been successfully processed and cannot be edited.
- This refund is currently being processed and cannot be edited.